reinforcement-learning/_posts/2018-12-17-tracto_td_learning.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: review
+title: "Tractography using reinforcement learning and adaptive-expanding graphs"
+tags: reinforcement-learning tractography
+author: "Antoine Théberge"
+cite:
+    authors: "Tingyi Wanyan, Lantao Liu, Eleftherios Garyfallidis"
+    title:   "Tractography using reinforcement learning and adaptive-expanding graphs"
+    venue:   "ISMRM 2015"
```

Comment:
Does the paper date back to ISMRM 2015?

Reply:
Woops, it was actually published at ISBI 2018